app/routes/downloaddetails.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, ResponseToolkit, ResponseObject } from "@hapi/hapi";
+import config from '../config'
+import * as utils from '../utils'
+
+module.exports = {
+  method: 'GET',
+  path: '/downloaddetails',
+  handler: async (_request: Request, h: ResponseToolkit) => {
```

Comment:
please remove the underscore from the variable. Its used to denote that the variable is not being used.

Reply:
ok, removed now